digestif/util.lua

```diff
@@ -391,8 +391,8 @@ local is_command_cmd = util.os_type == "windows"
 -- Return `name` if an executable with that name exists, nil
 -- otherwise.
 local function is_command(name)
-  local ok = os.execute(format(is_command_cmd, name))
-  return ok and name or nil
+  local ok, err = pcall(os.execute, format(is_command_cmd, name))
+  return err == 0 and ok and name or nil
```

Comment:
Why is this change needed? There is a difference between is_command_cmd exiting with non-zero status (which happens if a command named `name` doesn't exist) and the `os.execute` call failing, which presumably never happens.

Are you on Windows by any chance?